### PR TITLE
No speaker video warning

### PR DESCRIPTION
**Pull request by jstashh**
For #7 . Not sure if the design needs tweaking

### CODE REVIEW

File: Public/css/style.css [hunk 1]
```diff
@@ -449,6 +449,15 @@ section h2 {
     color: #616161;
 }
 
+/* no-content */
+
+.no-content-container {
```
Comment (ahmetws) [inline Public/css/style.css:453]: Could you please copy .form-container's styles to here? We used .form-container in the empty search text, it is better two of them become the same. You can check search.leaf file as a reference

File: Resources/Views/speaker.leaf [hunk 2]
```diff
@@ -25,6 +25,16 @@
 
 #embed("videosSection")
 
+#if(count(videos)==0) {
```
Comment (ahmetws) [inline Resources/Views/speaker.leaf]: Spaces needed before and after ==

**Review by ahmetws — CHANGES_REQUESTED** [review]
Sorry, I forgot to mention empty search text as a reference on this ticket. Can you please review the changes and send the pr again? Thanks for the contribution

**Comment by jstashh** [comment]
@ahmetws have made those changes, let me know if there is anything else

File: Resources/Views/speaker.leaf [hunk 3]
```diff
@@ -25,6 +25,18 @@
 
 #embed("videosSection")
 
+#if(count(videos) == 0) {
+<div class="container">
+    <div class="row">
+        <div class="col-sm">
+            <div class="form-container" style="text-align: center">
```
Comment (ahmetws) [inline Resources/Views/speaker.leaf]: Lets use the no-content-container instead of form-container. It is better to have no-content-container

**Review by ahmetws — CHANGES_REQUESTED** [review]

File: Resources/Views/speaker.leaf (hunk 3 above)
Comment (jstashh) [reply Resources/Views/speaker.leaf]: Ok, I've added .no-content-container back in and have made it copy the style from .form-container

**Review by ahmetws — APPROVED** [review]
👍

**Pull request merged**